test/Interop/Cxx/stdlib/Inputs/std-iterator.h

```diff
@@ -0,0 +1,25 @@
+#ifndef TEST_INTEROP_CXX_STDLIB_INPUTS_STD_ITERATOR_H
+#define TEST_INTEROP_CXX_STDLIB_INPUTS_STD_ITERATOR_H
+
+#include <iterator>
+#include <vector>
+
+using Iterator = std::vector<int>::iterator;
+
+inline Iterator initIterator() { return {}; }
+
+inline Iterator begin(std::vector<int> v) {
+  std::vector<int>::iterator v1 = v.begin();
+  return v1;
+}
```

Comment:
This is creating a dangling reference. Vector owns its storage, so it's going to be copied into `v`, and then you're getting a reference to the beginning of the vector, and the vector is destroyed before the reference (iterator) is used (by the caller).

Reply:
I see, will fix